docs/stream-aggregation.md

```diff
@@ -513,6 +513,8 @@ Below are aggregation functions that can be put in the `outputs` list at [stream
 * [count_series](#count_series)
 * [increase](#increase)
 * [increase_prometheus](#increase_prometheus)
+* [rate](#rate)
+* [rate_prometheus](#rate_prometheus)
```

Comment:
is this implemented?

Reply:
Do you refer to metricsql function? If yes, then answer is no: such function doesn't exist. The aggregation output won't exist as well.